src/agent/config.ts

```diff
@@ -357,6 +357,13 @@ export interface ResolvedDebugAgentConfig extends GoogleAuthOptions {
    * used to set a default api url
    */
   apiUrl?: string;
+
+  /**
+   * Number of times of the V8 breakpoint hits events before resetting (disable and
+   * re-enable) the breakpoints. This is to release the memory usage held by V8
+   * engine for each breakpoint hit to prevent the memory leak.
```

Comment:
Does this have a default value?  Could be nice to call out in the doc comment.

Reply:
Yeah there is a default value down in the bottom of this file. Will add it in the comment in next commit.